unixfs.go

```diff
@@ -70,8 +70,9 @@ type UnixfsAPI interface {
 	// Get returns a read-only handle to a file tree referenced by a path
 	//
 	// Note that some implementations of this API may apply the specified context
-	// to operations performed on the returned file
-	Get(context.Context, path.Path) (files.Node, error)
+	// to operations performed on the returned file.
+	// The bool parameter indicates whether token metadata should be returned.
+	Get(context.Context, path.Path, bool) (files.Node, error)
```

Comment:
@steveyeom For future, we should make this inside a new type `...options.UnixfsGetOption` to make new changes easier. Not urgent just a suggestion.